src/common.ts

```diff
@@ -1,10 +1,23 @@
 /**
- * Copyright (C) 2016-2018 Threema GmbH / SaltyRTC Contributors
+ * Copyright (C) 2016-2019 Threema GmbH / SaltyRTC Contributors
  *
  * Licensed under the Apache License, Version 2.0, <see LICENSE-APACHE file>
  * or the MIT license <see LICENSE-MIT file>, at your option. This file may not be
  * copied, modified, or distributed except according to those terms.
  */
-export class Common {
-    public static HEADER_LENGTH = 9;
+
+export const RELIABLE_ORDERED_HEADER_LENGTH = 1;
+export const UNRELIABLE_UNORDERED_HEADER_LENGTH = 9;
+
+/**
+ * The mode being used when chunking/unchunking.
+ */
```

Comment:
> It's required as a bitmask though, to clear out the reserved and unrelated bits.

Why? Regardless of whether or not you apply it, you can test with a logical and.

> I wonder if it would be better to provide a function that returns a `Mode`, instead of exposing those raw bitmasks that are easy to mishandle?

From the application's perspective these are just opaque modes. They coincidentally also map to the options bit field, so there's no function needed. I don't understand what's easy to mishandle here.

Reply:
> Maybe add a quick example?

:question: